pkg/config.go

```diff
@@ -39,6 +39,35 @@ func (*Config) Project() string {
 	return os.Getenv("GOOGLE_PROJECT")
 }
 
+// TlsVersion will be used current TLS Policy
+// by this value. The acceptable value MUST BE TLS 1.1/1.2/1.3
+// Note that the TLS 1.3 supports `TLS_AES_128_GCM_SHA256`,
+// `TLS_AES_256_GCM_SHA384` and `TLS_CHACHA20_POLY1305_SHA256` ciphers only.
+// Default is "TLS_1_2"
+func (*Config) TlsVersion() string {
+	tlsVersions := []string{"TLS_1_1", "TLS_1_2", "TLS_1_3"}
+	for _, ver := range tlsVersions {
+		if os.Getenv("MIN_TLS_VERSION") != ver || os.Getenv("MIN_TLS_VERSION") == "" {
+			return "TLS_1_2"
+		}
+	}
+	return os.Getenv("MIN_TLS_VERSION")
+}
+
+// SslProfile returns the string value of pre-configured profile
+// defined by GCP. The acceptable value MUST BE one of these values
+// COMPATIBLE/MODERN/RESTRICTED. We have not suppoerted CUSTOM value yet
+// Default is "MODERN"
+func (*Config) SslProfile() string {
+	sslProfiles := []string{"COMPATIBLE", "MODERN", "RESTRICTED"}
```

Comment:
There should be an easier way to check of a string is part of an array

Reply:
Since I'm not the Go expert guy for tuning this part. Let's wait for @Freyert comment to see whether there would have any better idea for this.

Reply:
We can use map but it's too much work for little improvement, I guess.  We can check if value match and return immediately instead, so it's be just a bit faster.

Reply:
I also think about the way using map but as you said it would take too much work. Using slice (array) seems like the proper way for this I think.